src/functionals.jl

```diff
@@ -1,34 +1,39 @@
-function getEndpoints(distr::UnivariateDistribution, epsilon::Real)
-    (left,right) = map(x -> quantile(distr,x), (0,1))
-    leftEnd = left!=-Inf ? left : quantile(distr, epsilon)
-    rightEnd = right!=-Inf ? right : quantile(distr, 1-epsilon)
-    (leftEnd, rightEnd)
-end
-
-function expectation(distr::ContinuousUnivariateDistribution, g::Function, epsilon::Real)
-    f = x->pdf(distr,x)
-    (leftEnd, rightEnd) = getEndpoints(distr, epsilon)
-    quadgk(x -> f(x)*g(x), leftEnd, rightEnd)[1]
+function expectation(distr::ContinuousUnivariateDistribution, g::Function; kwargs...)
```

Comment:
Can you deprecate `expectation(dist::ContinuosUnivariateDistribution, g::Function, epsilon::Real)` to ensure that the PR is not breaking?

Reply:
Just add it to src/deprecates.jl :slightly_smiling_face: